tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -56,8 +56,16 @@ sub run {
     }
 
     # Finally ensure that the screen sharing is available
```

Comment:
looks like this comment not valid after your patch . can you please update it ?

Reply:
done